### ISSUE TITLE
[Adaptive Cards Web-Main Section]: 'Link' present under 'Open your apps to extensibility' heading does not have discernible text defined.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7318]
https://microsoft.visualstudio.com/OS/_workitems/edit/37792669

**Comment by vagpt** [comment]
This issue is not repro, hence closing this issue.

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/323cfa71-3c57-4d97-a7ed-ecc14c831357)
